aten/src/ATen/native/mkldnn/xpu/detail/Matmul.cpp

```diff
@@ -8,6 +8,7 @@
 #include <Utils.h>
 
 #include <oneapi/dnnl/dnnl.hpp>
+#include "c10/core/ScalarType.h"
```

Comment:
```suggestion
#include <c10/core/ScalarType.h>
```

Reply:
Thanks for suggestions, the code is changed here